bom/openhab-addons/pom.xml

```diff
@@ -50,6 +50,11 @@
       <artifactId>org.openhab.binding.atlona</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.openhab.addons.bundles</groupId>
+      <artifactId>org.openhab.binding.ambientweather</artifactId>
```

Comment:
alphabet wise a little bit of :-)

Reply:
Duh! And English is my native language. LOL